.circleci/build_config.yml

```diff
@@ -240,9 +240,26 @@ step-depot-tools-get: &step-depot-tools-get
     name: Get depot tools
     command: |
       git clone --depth=1 https://chromium.googlesource.com/chromium/tools/depot_tools.git
-      # remove ninjalog_uploader_wrapper.py from autoninja since we don't use it and it causes problems
       if [ "`uname`" == "Darwin" ]; then
+        # remove ninjalog_uploader_wrapper.py from autoninja since we don't use it and it causes problems
         sed -i '' '/ninjalog_uploader_wrapper.py/d' ./depot_tools/autoninja
+        # Remove swift-format dep from cipd on macOS until we send a patch upstream.
+        cd depot_tools
+        patch gclient.py -p1 \<<'EOF'
+      diff --git a/gclient.py b/gclient.py
+      index c2a6fb3..f547b8e 100755
+      --- a/gclient.py
+      +++ b/gclient.py
+      @@ -673,7 +673,7 @@ class Dependency(gclient_utils.WorkItem, DependencySettings):
+      
+            if dep_type == 'cipd':
+              cipd_root = self.GetCipdRoot()
+      -        for package in dep_value.get('packages', []):
+      +        for package in (x for x in dep_value.get('packages', []) if "infra/3pp/tools/swift-format" not in x):
+                deps_to_add.append(
+                    CipdDependency(
+                        parent=self,
+      EOF
```

Comment:
can we maybe store this patch in a file instead of directly in the yml?

Reply:
@nornagon i chatted with @MarshallOfSound about this this morning - we figured it'd be less messy to do this in the short term